security-analysis/security-analysis-api/src/main/java/com/powsybl/security/limitreduction/LimitReductionDefinition.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.security.limitreduction;
+
+import com.google.common.collect.ImmutableList;
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.contingency.ContingencyContext;
+import com.powsybl.iidm.criteria.NetworkElementCriterion;
+import com.powsybl.iidm.criteria.duration.LimitDurationCriterion;
+import com.powsybl.iidm.network.LimitType;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * @author Olivier Perrin {@literal <olivier.perrin at rte-france.com>}
+ */
+public class LimitReductionDefinition {
+    private final LimitType limitType;
+    private final float limitReduction;
+    private final boolean monitoringOnly;
+    private final List<ContingencyContext> contingencyContexts;
```

Comment:
Not clear to me why we need a list of contingency context ?

Reply:
I used a list to manage the case where the reduction applies for several identified contingencies since there's no ContingencyContext dealing with multiple contigency IDs, but it allows unrealistic configurations.

I changed it to a unique ContingencyContext, but it will be best to add a ContingencyContext on multiple contigency IDs.